components/service/glean/src/main/java/mozilla/components/service/glean/net/HttpPingUploader.kt

```diff
@@ -93,10 +93,12 @@ internal class HttpPingUploader : PingUploader {
         config.pingTag?.let {
             headers.append("X-Debug-ID", it)
 
-            // NOTE: Tagged pings must be redirected to the GCP endpoint as the AWS endpoint isn't
-            // configured to handle them.  This may pose an issue with testing if a local server
-            // is used to capture pings.
-            endpoint = Configuration.DEFAULT_DEBUGVIEW_ENDPOINT
+            // NOTE: Tagged pings can be redirected to the GCP endpoint by prefixing the tag with
+            // `gcp-.  This may pose an issue with testing if a local server is used to capture
+            // pings.
+            if (it.startsWith("gcp-")) {
```

Comment:
Let's not overload the tag with specific meaning. If that's something we really need, let's add a separate option to the debug activity.

Reply:
This was really just a workaround due to Arkadiusz asking for the ability to configure the endpoint back to GCP if there were problems.  I would definitely like to remove the redirection altogether and just rely on the configuration endpoint.